Fastlane/Fastfile

```diff
@@ -190,3 +190,16 @@ lane :pr_changes_contains_path do |options|
 
   changes_contains_path
 end
+
+lane :clean_up_release_from_tag do |options|
+  latest_tag = ENV['BITRISE_GIT_TAG']
+  sh "git push origin --delete release/#{latest_tag}"
+  sh "git push origin --delete #{latest_tag}"
+end
+
+error do |lane, exception|
+  # Makes sure we clean up the tag and the release branch if release_from_tag failed, to allow future releases
+  if lane.eql?(:release_from_tag)
+    clean_up_release_from_tag
```

Comment:
I wondered whether this would be redundant since we have the Bitrise workflow step, but I feel it's good to keep it in place for scenarios like failing local runs 👌

Reply:
I thought about that as well and came to the same conclusion :B